src/components/Dashboard/Tiles/TotalUsers/TotalUsers.tsx

```diff
@@ -2,25 +2,34 @@ import React, { ReactElement } from 'react';
 import { connect } from 'react-redux';
 import { RootState } from '../../../../store';
 
-import { CountTitle, NumberDisplay, TileContainer } from './styled';
+import {
+  CountTitle,
+  MaxCapacity,
+  NumberDisplay,
+  TileContainer,
+} from './styled';
 
-interface TotalUsersProps {
-  count: number;
+interface CurrentUserCapacityProp {
+  maxCapacity: number;
+  currentPopulation: number;
 }
 
 const mapStateToProps = (state: RootState) => {
   return {
-    count: state.person.size,
+    maxCapacity: state.facility.capacity,
+    currentPopulation: state.facility.currentPopulation,
   };
 };
 
-const TotalUsers: React.FC<TotalUsersProps> = (
-  props: TotalUsersProps,
+const TotalUsers: React.FC<CurrentUserCapacityProp> = (
+  props: CurrentUserCapacityProp,
 ): ReactElement => {
   return (
     <TileContainer>
+      <MaxCapacity>Max Capacity</MaxCapacity>
+      <NumberDisplay>{props.maxCapacity}</NumberDisplay>
       <CountTitle>Total Users</CountTitle>
```

Comment:
Change this to "Current Population"

Reply:
<img width="1792" alt="Screen Shot 2021-02-21 at 12 26 46 PM" src="https://user-images.githubusercontent.com/43226793/108633048-17259500-7440-11eb-9f25-3f8031f1a69e.png">

voila now onto the styling